README.md

```diff
@@ -5,7 +5,22 @@ The full PyTOM repository can be found at: https://github.com/SBC-Utrecht/PyTom
 
 ## Citation
 
-...
+For a reference on GPU accelerated template matching in tomograms please see the [IJMS publication](https://www.mdpi.com/1422-0067/24/17/13375).
+
+
+@Article{ijms241713375,
+AUTHOR = {Chaillet, Marten L. and van der Schot, Gijs and Gubins, Ilja and Roet, Sander and Veltkamp, Remco C. and Förster, Friedrich},
+TITLE = {Extensive Angular Sampling Enables the Sensitive Localization of Macromolecules in Electron Tomograms},
+JOURNAL = {International Journal of Molecular Sciences},
+VOLUME = {24},
+YEAR = {2023},
+NUMBER = {17},
+ARTICLE-NUMBER = {13375},
+URL = {https://www.mdpi.com/1422-0067/24/17/13375},
+ISSN = {1422-0067},
+DOI = {10.3390/ijms241713375}
+}
```

Comment:
please put this in code block so the formatting is not all broken:
````suggestion
```
@Article{ijms241713375,
AUTHOR = {Chaillet, Marten L. and van der Schot, Gijs and Gubins, Ilja and Roet, Sander and Veltkamp, Remco C. and Förster, Friedrich},
TITLE = {Extensive Angular Sampling Enables the Sensitive Localization of Macromolecules in Electron Tomograms},
JOURNAL = {International Journal of Molecular Sciences},
VOLUME = {24},
YEAR = {2023},
NUMBER = {17},
ARTICLE-NUMBER = {13375},
URL = {https://www.mdpi.com/1422-0067/24/17/13375},
ISSN = {1422-0067},
DOI = {10.3390/ijms241713375}
}
```
````